### PR TITLE
Added docs for cEUR listing

### LINKED ISSUE #7146: Update Developer Guide > Integration > Listings to also reference cEUR
Make this page more generic such that we have to minimally change it for future stablecoins: https://docs.celo.org/developer-guide/integrations/listings

### PR DESCRIPTION
**Pull request by martinvol**
### Description

Docs update

### Other changes

-

### Tested

- 

### Related issues

- Closes #7146

### CODE REVIEW

File: packages/docs/developer-resources/integrations/listings.md [hunk 1]
```diff
@@ -20,45 +20,52 @@ A general overview of integrations that would be relevant to you listing Celo Pl
...
 
 For API endpoints useful for listing that follow [CMC requirements](https://docs.google.com/document/d/1S4urpzUnO2t7DmS_1dc4EL4tgnnbTObPYXvDeBnukCg/edit#)
 
+##### Mento Addresses
+* cUSD/CELO contract - [`0x67316300f17f063085Ca8bCa4bd3f7a5a3C66275`](https://explorer.celo.org/address/0x67316300f17f063085Ca8bCa4bd3f7a5a3C66275/transactions)
+* cEUR/CELO contract - [`0xE383394B913d7302c49F794C7d3243c429d53D1d`](https://explorer.celo.org/address/0xE383394B913d7302c49F794C7d3243c429d53D1d/transactions)
+
 ##### Summary
-Summary overview of market data for all tickers and all markets.
+Summary overview of market data for all tickers and all markets. This endpoints don't yet support cEUR.
```
Comment (pedro-clabs) [inline packages/docs/developer-resources/integrations/listings.md]: Is it just the Summary endpoint or all of them that don't support cEUR?

Comment (yerdua) [inline packages/docs/developer-resources/integrations/listings.md]: small grammar fix for plural/singular mismatch:

_if it's a single endpoint (looks like it)_
"This **endpoint doesn't** yet support cEUR."

_multiple endpoints_
"**These** endpoints don't yet support cEUR."

Or, you could reword it and avoid the issue entirely:
"cEUR is not yet supported."

**Review by yerdua — APPROVED** [review]
One suggested grammar correction. Otherwise, LGTM.

File: packages/docs/developer-resources/integrations/listings.md (hunk 1 above)
Comment (martinvol) [inline packages/docs/developer-resources/integrations/listings.md]:
```suggestion
Summary overview of market data for all tickers and all markets. These endpoints don't yet support cEUR.
```

**Pull request merged**